end-to-end-computer-vision/README.md

```diff
@@ -63,37 +64,52 @@ zenml connect --url <INSERT_ZENML_URL_HERE>
 We will use GCP in the commands listed below, but it will work for other cloud
 providers.
 
-### Follow our guide to set up your credential for GCP
+1) Follow our guide to set up your credentials for GCP [here](https://docs.zenml.io/how-to/auth-management/gcp-service-connector)
 
-[Set up a GCP service
-connector](https://docs.zenml.io/how-to/auth-management/gcp-service-connector)
+2) Set up a bucket in GCP to persist your training data
 
-### Set up a bucket to persist your training data
-
-### Set up a bucket to use as artifact store within ZenML
-
-[Learn how to set up a GCP artifact store stack component within zenml
-here](https://docs.zenml.io/stack-components/artifact-stores)
-### Set up vertex for pipeline orchestration
-
-[Learn how to set up a Vertex orchestrator stack component within zenml
-here](https://docs.zenml.io/stack-components/orchestrators/vertex)
-### For training on accelerators like GPUs/TPUs set up Vertex
-
-[Learn how to set up a Vertex step operator stack component within zenml
-here](https://docs.zenml.io/stack-components/step-operators/vertex)
-### Set up Container Registry
-
-[Learn how to set up a google cloud container registry component within zenml
-here](https://docs.zenml.io/stack-components/container-registries/gcp)
+3) Set up a bucket to use as artifact store within ZenML
+Learn how to set up a GCP artifact store stack component within zenml
+[here](https://docs.zenml.io/stack-components/artifact-stores)
+4) Set up vertex for pipeline orchestration
+Learn how to set up a Vertex orchestrator stack component within zenml
+[here](https://docs.zenml.io/stack-components/orchestrators/vertex)
+5) For training on accelerators like GPUs/TPUs set up Vertex
+Learn how to set up a Vertex step operator stack component within zenml
+[here](https://docs.zenml.io/stack-components/step-operators/vertex)
+6) Set up Container Registry in GCP. Learn how to set up a google cloud container registry component within zenml
+[here](https://docs.zenml.io/stack-components/container-registries/gcp)
 
 ## Label Studio
 
-### [Start Label Studio locally](https://labelstud.io/guide/start)
-### [Follow these ZenML instructions to set up Label Studio as a stack component](https://docs.zenml.io/stack-components/annotators/label-studio)
-### Create a project within Label Studio and name it `ship_detection_gcp`
-### [Set up Label Studio to use external storage](https://labelstud.io/guide/storage) 
-use the first bucket that you created to data persistence
+1) [Start Label Studio locally](https://labelstud.io/guide/start)
+For label studio we recommend using docker/docker-compose to deploy a local instance
+```bash
+zenml integration install label_studio
```

Comment:
```suggestion
```

No need for this here if you did it above already.

Reply:
good point